roles/image_builder/README.md

```diff
@@ -316,6 +316,18 @@ microshift_image_crio_proxy:
   port: 123
 ```
 
+### microshift_image_pull_secret
+
+Type: file / string
+Required: false
+
+Pull secret allows authentication with the container registries that serve the container images used by MicroShift.
```

Comment:
I think we should mention that this is for the product and not the upstream bits. Also, I think it's a good idea to link to the Red Hat docs on the image pull secret for added context.

Reply:
Sounds good, I updated the readme. Hopefully that clarifies the variable better.